src/types/errors.ts

```diff
@@ -0,0 +1,9 @@
+export type ModuleNotFoundError = Error & {
+    code: 'MODULE_NOT_FOUND'
+}
+
+export const isModuleNotFoundError = (
```

Comment:
I'm a bit unsure if `type-guards` should live with the types, as they are "runtime" code. But it feels very convenient.

Reply:
I've moved them to `lib` now :)